google-cloud-logging/lib/google/cloud/logging/logger.rb

```diff
@@ -307,26 +318,26 @@ def level= severity
         alias_method :sev_threshold=, :level=
 
         ##
-        # Track a given trace_id by associating it with the current
-        # Thread
+        # Associate request data with the current Thread
         #
-        # @param [String] trace_id The HTTP_X_CLOUD_TRACE_CONTEXT HTTP request
-        #   header that's shared and tracked by all Stackdriver services
-        def add_trace_id trace_id
-          trace_ids[current_thread_id] = trace_id
+        # @param [RequestInfo] info Info about the current request
+        def add_request_info info
```

Comment:
Yeah, I think I'm going to ditch the RequestInfo object for the moment and just have the two fields separately. This change used to be somewhat bigger, keeping track of four separate fields rather than two, but I pared it down and now I'm not convinced we need to expose the object at all.